### PR TITLE
match Add() and AddRepos() and correct typos

### PR DESCRIPTION
**Pull request by julienduchesne**
Supersedes https://github.com/grafana/tanka/pull/640 (adding unit tests)

### CODE REVIEW

File: pkg/helm/charts.go [hunk 1]
```diff
@@ -144,13 +144,9 @@ func (c Charts) Vendor() error {
 
 // Add adds every Chart in reqs to the Manifest after validation, and runs
 // Vendor afterwards
-func (c *Charts) Add(reqs []string) error {
+func (c *Charts) Add(reqs []string, vendor bool) error {
```
Comment (sh0rez) [inline pkg/helm/charts.go:147]: What's the use of that bool argument? Like, what a case where I like to add but not vendor?

Comment (julienduchesne) [reply pkg/helm/charts.go:147]: Tests, but I think I'll setup a fake local helm repo instead. I just don't want to depend on a remote helm chart for tests

Comment (julienduchesne) [reply pkg/helm/charts.go]: Just noticed that there was already a helm interface there. Much better

**Comment by redradrat** [comment]
@julienduchesne thx for helping out my lazy ass with the tests here ;)

**Review by jvrplmlmn — APPROVED** [review]
👍

**Pull request merged**